tests/test_dialects.py

```diff
@@ -307,6 +307,70 @@ def test_starrocks(self):
             write="starrocks",
         )
 
+    def test_bigquery(self):
+        self.validate(
+            "SELECT CAST(`a`.`b` AS INT) FROM foo",
+            "SELECT CAST(`a`.`b` AS INT64) FROM foo",
+            read="bigquery",
+            write="bigquery",
+        )
+
+        self.validate(
```

Comment:
you didn’t test any inverses can you read bigquery int64 and output generic?

Reply:
That's a good point, I can add that, had some questions:

- What's generic `sqlglot` SQL? Just leave `write=` unspecified?
- When I add `read='bigquery'` using ```"SELECT CAST(`a`.`b` AS INT64) FROM foo",```, I get this error:

```sql
sqlglot.errors.ParseError: Expected TYPE after CAST. Line 1, Col: 24.
  SELECT CAST(`a`.`b` AS INT64) FROM foo
```

I'm not sure how to modify my Dialect class to support the inverse operation, all of the `type_mapping` class attributes have the generic `exp.DataType.Type.*` on the left-hand side and the string on the right-hand side. Do you have any suggestions for me?

Reply:
yes, just leave write unspecified, it should automatically support the inverse